src/config.rs

```diff
@@ -78,6 +78,8 @@ create_config! {
     imports_indent: IndentStyle, IndentStyle::Block, false, "Indent of imports";
     imports_layout: ListTactic, ListTactic::Mixed, false, "Item layout inside a import block";
     merge_imports: bool, false, false, "Merge imports";
+    group_imports: GroupImportsTactic, GroupImportsTactic::Preserve, false,
+        "Reorganize import groups";
```

Comment:
Minor nit but could we update this with something similar to what was changed in the Configurations.md file, like `Controls the strategy for how imports are grouped together`?